src/dialogevent.h

```diff
@@ -13,15 +13,15 @@ class DialogEvent : public QDialog
     Q_OBJECT
 
 public:
-    explicit DialogEvent(QWidget *parent = 0);
+    explicit DialogEvent(QWidget *parent = nullptr);
     ~DialogEvent();
 
     static int edit(QWidget *parent, RPG::Event *event);
 
     static bool equalEvents(const RPG::Event &e1,
                             const RPG::Event &e2);
 
-    RPG::Event event() const;
+    RPG::Event getEvent() const;
```

Comment:
any reason this was renamed? At least just lowercase without get-prefix is the Qt-convention for the getter.
Do we use get-prefixed getters in the other parts of the editor?

Reply:
There's some conflict with some Qt member called ::event if kept :confused: